charmcraft/commands/store/__init__.py

```diff
@@ -54,7 +54,7 @@ def run(self, parsed_args):
 class WhoamiCommand(BaseCommand):
     """Show login information."""
     name = 'whoami'
-    help_msg = "returns your login information relevant to the store"
+    help_msg = "returns your login information relevant to the Store"
```

Comment:
It feels too strong to use a Proper Noun.
```
$ snapcraft --help
...
  export-login       Save login configuration for a store account in FILE.
...
  whoami             Returns your login information relevant to the store.
```

I wouldn't capitalize them unless there is a design guideline that says we should.
That said, we probably *should* use full Sentence case (with final stop).
So:
```
help_msg = "Returns your login information relevant to the store."
```

Reply:
That initial capitalization is still on discussion (particularly, I asked about these details to Lilyana, she said still to be defined).

I can go back to lowecase `store`, I just wanted to emphasize that is not "any store", but "THE Store".